# Vocabulary Level Analysis

In [2]:
import pandas as pd
kor_xls = pd.read_excel('kor_words.xls', index_col=None)
kor_xls.to_csv('kor_words.csv', encoding='utf-8', index=False)

In [3]:
kor = pd.read_csv('kor_words.csv')

In [17]:
kor.head()

순위     단어 품사   풀이 등급
0  1195.0     가게  명  NaN  A
1   898.0   가격03  명   價格  B
2  2986.0   가구03  명   家口  C
3  7434.0   가구04  명   家具  B
4  4964.0  가까워지다  동  NaN  B

In [30]:
kor = kor.drop(columns=['풀이', '순위'])
kor.head()

단어 품사 등급
0     가게  명  A
1   가격03  명  B
2   가구03  명  C
3   가구04  명  B
4  가까워지다  동  B

- A등급, 1단계, 가장 쉬움
- B등급, 2단계, 중간
- C등급, 3단계, 어려움

### Komoran 기준
- 감 : 감탄사 IC 
- 고 : 고유 명사 NNP	
- 관 : 관형사 MM
- 대 : 대명사 NP
- 동 : 동사 VV
- 명 : 명사 NNG
- 보 : 보조 용언	VX
- 부 : 부사 MAG
- 불 : 분석 불능 -> 제거하기
- 수 : 수사 NR
- 의 : 의존 명사	NNB
- 형 : 형용사 VA

In [31]:
kor = kor.replace(['감', '고', '관', '대', '동', '명', '보', '부', '불', '수', '의', '형'], ['IC', 'NNP', 'MM', 'NP', 'VV', 'NNG', 'VX', 'MAG', 'None', 'NR', 'NNB', 'VA'])

In [22]:
kor.head()

단어   품사 등급
0     가게  NNG  A
1   가격03  NNG  B
2   가구03  NNG  C
3   가구04  NNG  B
4  가까워지다   VV  B

In [32]:
kor = kor.replace(['A', 'B', 'C'], [1,2,3])
kor.head()

단어   품사  등급
0     가게  NNG   1
1   가격03  NNG   2
2   가구03  NNG   3
3   가구04  NNG   2
4  가까워지다   VV   2

In [33]:
kor['단어'] = kor['단어'].str.replace('\d+', '')
kor.head()

단어   품사  등급
0     가게  NNG   1
1     가격  NNG   2
2     가구  NNG   3
3     가구  NNG   2
4  가까워지다   VV   2

In [34]:
kor.to_csv('clean_kor.csv', index=False, encoding='utf-8')

In [4]:
kor = pd.read_csv('clean_kor.csv')

In [5]:
with open('netflix.txt', 'r', encoding='UTF8') as f:
    lines = f.read().splitlines()
lines

['좀비가 된 의대생은 인간성을 유지하기 위해 영안실에서 뇌를 먹어치우며 지내던 중 범죄를 해결하는 묘한 재능이 생겼음을 발견한다.',
 '치명적인 좀비 바이러스에서 살아남은 남자가 백신에 대한 키를 쥐고 있다고 믿고, 그들은 위험천만한 미국 횡단에 도전한다.',
 '어머니가 죽은 후 연주에 대한 의지를 잃어버린 피아노 신동. 그러나 바이올린을 연주하는 자유분방한 성격의 소녀를 만나면서 그의 삶은 달라진다.',
 '극한의 기후환경 속에서도 다양한 생물체들이 더불어 살아가는 천혜의 땅, 알래스카의 세 계절은 저마다 다양한 축복과 도전을 선사한다.',
 '냉혹한 상속녀 이경, 그녀를 사랑한 남자 건우, 이경이 되고 싶은 여자 세진. ‘권력과 금력의 용광로’ 속으로 부나비처럼 뛰어드는 세 남녀의 이야기가 펼쳐진다.',
 '성공을 위해 자신만을 믿고 살아온 폭력배 출신의 차가운 남자 한태상은 당돌한 여자 미도에게 마음이 흔들린다.',
 '1900년대 초, 안락한 상류층 도시 생활을 뒤로하고 캐나다 서부의 탄광촌에서 교사로 사는 삶을 택한 씩씩한 여성의 이야기.',
 '게으르지만 매력적인 경찰 잭 암스트롱과 의욕 넘치고 똑똑한 경할 케이트 비숍이 서로에게 맞춰가며 함께 사건을 해결해 나간다.',
 '프랑스의 루이 14세가 건설을 명한 호화로운 궁전은 사랑과 권력을 향한 암투가 벌어지는 치열한 전쟁터이자 귀족들을 통제하기 위한 금박을 두른 감옥이 된다.',
 '숫자의 패턴을 파악하는 데 천부적인 재능이 있는 자폐아 소년은 비록 말을 하지는 못하지만 미래 사건을 예측하는 능력이 있는 것으로 보인다.',
 '약과 갱의 법칙이 지배하는 이스트 런던 빈민가. 마약을 팔거나 아니면 죽거나, 누구도 안심할 수 없다. 3년 동안 빼앗겼던 왕좌를 되찾으러 두셰인이 돌아온다.',
 '청소년의 몸을 한 뱀파이어 스테판과 데이먼은 서로 앙숙으로, 엘레나라는 소녀를 사로잡기 위해 경쟁한다.',
 '스물의 봄, 스물일곱의 여름, 그리고 서른둘의 가을과 겨울. 그렇게 두 남녀는 세 번째 만났다.

In [8]:
from konlpy.tag import Komoran 
tagger = Komoran()
tagged_sentences = [tagger.pos(sent) for sent in sentences]

In [9]:
tagged_sentences

[[('좀비', 'NNP'),
  ('가', 'JKS'),
  ('되', 'VV'),
  ('ㄴ', 'ETM'),
  ('의대', 'NNG'),
  ('생', 'XSN'),
  ('은', 'JX'),
  ('인간성', 'NNP'),
  ('을', 'JKO'),
  ('유지', 'NNG'),
  ('하', 'XSV'),
  ('기', 'ETN'),
  ('위하', 'VV'),
  ('아', 'EC'),
  ('영안실', 'NNG'),
  ('에서', 'JKB'),
  ('뇌', 'NNG'),
  ('를', 'JKO'),
  ('먹', 'VV'),
  ('어', 'EC'),
  ('치우', 'VX'),
  ('며', 'EC'),
  ('지내', 'VV'),
  ('던', 'ETM'),
  ('중', 'NNB'),
  ('범죄', 'NNG'),
  ('를', 'JKO'),
  ('해결', 'NNG'),
  ('하', 'XSV'),
  ('는', 'ETM'),
  ('묘하', 'VA'),
  ('ㄴ', 'ETM'),
  ('재능', 'NNG'),
  ('이', 'JKS'),
  ('생기', 'VV'),
  ('었', 'EP'),
  ('음', 'ETN'),
  ('을', 'JKO'),
  ('발견', 'NNG'),
  ('하', 'XSV'),
  ('ㄴ다', 'EF'),
  ('.', 'SF')],
 [('치명', 'NNG'),
  ('적', 'XSN'),
  ('이', 'VCP'),
  ('ㄴ', 'ETM'),
  ('좀비', 'NNP'),
  ('바이러스', 'NNP'),
  ('에서', 'JKB'),
  ('살아남', 'VV'),
  ('은', 'ETM'),
  ('남자', 'NNG'),
  ('가', 'JKS'),
  ('백신', 'NNP'),
  ('에', 'JKB'),
  ('대하', 'VV'),
  ('ㄴ', 'ETM'),
  ('키', 'NNG'),
  ('를', 'JKO'),
  ('쥐', 'VV'),
  ('고', 'EC'),
  ('있', 'VV'

In [10]:
netflix = []
for sent in tagged_sentences:    
    for word, tag in sent:
        if tag in ['VV', 'VA']:
            netflix.append(word)
            
netflix = [v+'다' for v in netflix]

In [11]:
netflix

['되다',
 '위하다',
 '먹다',
 '지내다',
 '묘하다',
 '생기다',
 '살아남다',
 '대하다',
 '쥐다',
 '있다',
 '믿다',
 '죽다',
 '대하다',
 '잃어버리다',
 '만나다',
 '달라지다',
 '더불다',
 '살아가다',
 '되다',
 '뛰어들다',
 '펼쳐지다',
 '위하다',
 '믿다',
 '살아오다',
 '차갑다',
 '흔들리다',
 '뒤로하다',
 '살다',
 '택하다',
 '게으르다',
 '넘치다',
 '트다',
 '맞추다',
 '향하다',
 '벌어지다',
 '위하다',
 '두르다',
 '되다',
 '있다',
 '하다',
 '있다',
 '보이다',
 '팔다',
 '죽다',
 '없다',
 '빼앗기다',
 '되찾다',
 '돌아오다',
 '사로잡다',
 '위하다',
 '그렇다',
 '만나다',
 '보이다',
 '가지다',
 '빠지다',
 '길다',
 '그리다',
 '드리우다',
 '되다',
 '빼앗기다',
 '흩어지다',
 '찾다',
 '떠나다',
 '되찾다',
 '귀엽다',
 '돌아오다',
 '어리다',
 '얕다',
 '크다',
 '치다',
 '거리다',
 '다투다',
 '하다',
 '나타나다',
 '뭉치다',
 '만나다',
 '그렇다',
 '더하다',
 '깨닫다',
 '되다',
 '들르다',
 '알다',
 '되다',
 '지키다',
 '위하다',
 '싸우다',
 '위하다',
 '치다',
 '마시다',
 '꿈꾸다',
 '모이다',
 '되다',
 '위하다',
 '나아가다',
 '펼쳐지다',
 '떼다',
 '없다',
 '위하다',
 '붙잡다',
 '되다',
 '되다',
 '담다',
 '깨어나다',
 '놀랍다',
 '갖다',
 '되다',
 '맞서다',
 '싸우다',
 '떨어지다',
 '알다',
 '바뀌다',
 '살아가다',
 '되다',
 '잡다',
 '먹다',
 '살다',
 '위하다',
 '흐르다',
 '살다',
 '알아내다',
 '위하다',
 '보내다',
 '많다',
 '닥치다',
 '선보이다',
 '막다',
 '위하다',
 

In [12]:
for sent in tagged_sentences:    
    for word, tag in sent:
        if tag in ['IC', 'NNP', 'MM', 'NP', 'NNG', 'VX', 'MAG', 'None', 'NR', 'NNB']:
            netflix.append(word)

In [13]:
netflix

['되다',
 '위하다',
 '먹다',
 '지내다',
 '묘하다',
 '생기다',
 '살아남다',
 '대하다',
 '쥐다',
 '있다',
 '믿다',
 '죽다',
 '대하다',
 '잃어버리다',
 '만나다',
 '달라지다',
 '더불다',
 '살아가다',
 '되다',
 '뛰어들다',
 '펼쳐지다',
 '위하다',
 '믿다',
 '살아오다',
 '차갑다',
 '흔들리다',
 '뒤로하다',
 '살다',
 '택하다',
 '게으르다',
 '넘치다',
 '트다',
 '맞추다',
 '향하다',
 '벌어지다',
 '위하다',
 '두르다',
 '되다',
 '있다',
 '하다',
 '있다',
 '보이다',
 '팔다',
 '죽다',
 '없다',
 '빼앗기다',
 '되찾다',
 '돌아오다',
 '사로잡다',
 '위하다',
 '그렇다',
 '만나다',
 '보이다',
 '가지다',
 '빠지다',
 '길다',
 '그리다',
 '드리우다',
 '되다',
 '빼앗기다',
 '흩어지다',
 '찾다',
 '떠나다',
 '되찾다',
 '귀엽다',
 '돌아오다',
 '어리다',
 '얕다',
 '크다',
 '치다',
 '거리다',
 '다투다',
 '하다',
 '나타나다',
 '뭉치다',
 '만나다',
 '그렇다',
 '더하다',
 '깨닫다',
 '되다',
 '들르다',
 '알다',
 '되다',
 '지키다',
 '위하다',
 '싸우다',
 '위하다',
 '치다',
 '마시다',
 '꿈꾸다',
 '모이다',
 '되다',
 '위하다',
 '나아가다',
 '펼쳐지다',
 '떼다',
 '없다',
 '위하다',
 '붙잡다',
 '되다',
 '되다',
 '담다',
 '깨어나다',
 '놀랍다',
 '갖다',
 '되다',
 '맞서다',
 '싸우다',
 '떨어지다',
 '알다',
 '바뀌다',
 '살아가다',
 '되다',
 '잡다',
 '먹다',
 '살다',
 '위하다',
 '흐르다',
 '살다',
 '알아내다',
 '위하다',
 '보내다',
 '많다',
 '닥치다',
 '선보이다',
 '막다',
 '위하다',
 

In [14]:
easy = kor.query('등급==1')
easy.head()

단어   품사  등급
0    가게  NNG   1
7   가깝다   VA   1
9    가끔  MAG   1
17   가다   VV   1
18   가다   VX   1

In [58]:
easy.query('단어=="한"')

단어  품사  등급
5660  한  MM   1

In [15]:
easy_words = easy['단어'].tolist()

In [48]:
easy_list=[]
easy_count =0
for word in naver:
    if word in easy_words:
        easy_list.append(word)
        easy_count+=1

In [85]:
len(netflix)

26508

In [17]:
easy_list

['되다',
 '먹다',
 '지내다',
 '생기다',
 '있다',
 '죽다',
 '잃어버리다',
 '만나다',
 '되다',
 '살다',
 '되다',
 '있다',
 '하다',
 '있다',
 '팔다',
 '죽다',
 '없다',
 '돌아오다',
 '그렇다',
 '만나다',
 '가지다',
 '길다',
 '그리다',
 '되다',
 '찾다',
 '떠나다',
 '돌아오다',
 '크다',
 '치다',
 '하다',
 '만나다',
 '그렇다',
 '되다',
 '알다',
 '되다',
 '싸우다',
 '치다',
 '마시다',
 '되다',
 '없다',
 '되다',
 '되다',
 '되다',
 '싸우다',
 '알다',
 '되다',
 '잡다',
 '먹다',
 '살다',
 '살다',
 '보내다',
 '많다',
 '있다',
 '하다',
 '다니다',
 '되다',
 '들어가다',
 '걸다',
 '들다',
 '싸우다',
 '싸우다',
 '잃다',
 '받다',
 '어떻다',
 '만나다',
 '되다',
 '있다',
 '떠나다',
 '만나다',
 '많다',
 '많다',
 '만나다',
 '만나다',
 '잊다',
 '아프다',
 '잊다',
 '죽다',
 '열다',
 '죽다',
 '되다',
 '들다',
 '즐겁다',
 '잊다',
 '하다',
 '되다',
 '받다',
 '되다',
 '살다',
 '오르다',
 '걸다',
 '없다',
 '어떻다',
 '만들다',
 '쓰다',
 '되다',
 '없다',
 '좋아하다',
 '알다',
 '되다',
 '하다',
 '보다',
 '오다',
 '나가다',
 '되다',
 '찾다',
 '되다',
 '높다',
 '가다',
 '배우다',
 '쓰다',
 '들어가다',
 '멀다',
 '받다',
 '작다',
 '높다',
 '아름답다',
 '되다',
 '입다',
 '하다',
 '아름답다',
 '잃다',
 '찾다',
 '만나다',
 '돌아가다',
 '바꾸다',
 '있다',
 '있다',
 '보내다',
 '하다',
 '받다',
 '피우다',
 '하다',
 '있다',
 '바꾸다',
 '있다',
 

In [49]:
from collections import Counter
Counter(easy_list).most_common()

[('되다', 1727),
 ('그', 1520),
 ('하다', 802),
 ('것', 798),
 ('이', 782),
 ('있다', 775),
 ('수', 675),
 ('없다', 551),
 ('한', 533),
 ('시작', 504),
 ('분', 480),
 ('사람', 377),
 ('주', 372),
 ('알다', 365),
 ('받다', 356),
 ('사랑', 340),
 ('일', 311),
 ('년', 291),
 ('함께', 281),
 ('말', 281),
 ('날', 279),
 ('만나다', 273),
 ('친구', 269),
 ('찾다', 268),
 ('속', 254),
 ('두', 251),
 ('보다', 248),
 ('살다', 244),
 ('전', 242),
 ('모든', 241),
 ('어느', 218),
 ('오', 203),
 ('가다', 195),
 ('이야기', 189),
 ('가족', 188),
 ('후', 185),
 ('남자', 183),
 ('명', 177),
 ('다시', 177),
 ('같다', 174),
 ('오다', 173),
 ('크다', 171),
 ('집', 165),
 ('떠나다', 160),
 ('더', 160),
 ('다른', 154),
 ('안', 149),
 ('만', 148),
 ('그렇다', 147),
 ('여자', 146),
 ('만들다', 144),
 ('앞', 141),
 ('아버지', 140),
 ('시간', 136),
 ('딸', 136),
 ('세', 135),
 ('마음', 135),
 ('번', 134),
 ('아들', 132),
 ('모르다', 130),
 ('아내', 130),
 ('이제', 128),
 ('가지다', 127),
 ('엄마', 127),
 ('보내다', 123),
 ('가장', 121),
 ('때', 121),
 ('죽다', 117),
 ('모두', 117),
 ('둘', 114),
 ('하나', 114),
 ('돌아오다', 111),
 ('바로'

In [50]:
naver_easy_count = str(Counter(easy_list).most_common())

In [51]:
with open('naver_easy_count .txt', 'w', encoding='UTF8') as f:
         f.write(naver_easy_count)

In [18]:
easy_count

8951

In [86]:
# 넷플릭스 1단계 단어 비율 
easy_count / len(netflix)

0.337671646295458

In [24]:
mid_words = kor.query('등급==2')['단어'].tolist()

In [52]:
mid_list=[]
mid_count =0
for word in naver:
    if word in mid_words:
        mid_list.append(word)
        mid_count+=1

In [53]:
from collections import Counter
Counter(mid_list).most_common()

[('위하다', 665),
 ('자신', 619),
 ('지', 485),
 ('말', 281),
 ('중', 251),
 ('세계', 221),
 ('오', 203),
 ('세상', 200),
 ('빠지다', 195),
 ('대하다', 160),
 ('사실', 159),
 ('비밀', 154),
 ('만', 148),
 ('마을', 141),
 ('최고', 137),
 ('향하다', 132),
 ('한편', 128),
 ('발견', 123),
 ('구하다', 120),
 ('죽다', 117),
 ('죽음', 117),
 ('없이', 116),
 ('나타나다', 114),
 ('통하다', 112),
 ('인간', 112),
 ('지구', 112),
 ('전쟁', 111),
 ('등', 111),
 ('결국', 111),
 ('느끼다', 105),
 ('서로', 104),
 ('뿐', 103),
 ('순간', 102),
 ('인생', 101),
 ('아이', 98),
 ('지키다', 97),
 ('믿다', 97),
 ('사고', 97),
 ('사라지다', 95),
 ('우연히', 95),
 ('점점', 95),
 ('단', 92),
 ('의하다', 90),
 ('보이다', 89),
 ('성공', 89),
 ('밤', 88),
 ('어리다', 87),
 ('간', 86),
 ('소녀', 85),
 ('과거', 85),
 ('찾아오다', 84),
 ('자', 84),
 ('나서다', 79),
 ('마침내', 79),
 ('들다', 77),
 ('소년', 75),
 ('대', 74),
 ('벌어지다', 73),
 ('죽이다', 73),
 ('쓰다', 73),
 ('드라마', 71),
 ('얻다', 70),
 ('살', 70),
 ('그런', 70),
 ('갖다', 69),
 ('나다', 68),
 ('모습', 67),
 ('도움', 67),
 ('선수', 67),
 ('남', 66),
 ('뉴욕', 63),
 ('능력', 63),
 ('첫', 63),
 ('적', 6

In [54]:
naver_mid_count = str(Counter(mid_list).most_common())

In [55]:
with open('naver_mid_count .txt', 'w', encoding='UTF8') as f:
         f.write(naver_mid_count)

In [29]:
mid_count

5975

In [91]:
# 넷플릭스 2단계 단어 비율 
mid_count / len(netflix)

0.2254036517277803

In [56]:
diff_words = kor.query('등급==3')['단어'].tolist()

diff_list=[]
diff_count =0
for word in naver:
    if word in diff_words:
        diff_list.append(word)
        diff_count+=1

In [33]:
diff_count

5339

In [57]:
Counter(diff_list).most_common()
naver_diff_count = str(Counter(diff_list).most_common())
with open('naver_diff_count .txt', 'w', encoding='UTF8') as f:
         f.write(naver_diff_count)

In [94]:
# 넷플릭스 3단계 단어 비율 
diff_count / len(netflix)

0.20141089482420402

## 넷플릭스
- 1단계 : 33.7%
- 2단계 : 22.5%
- 3단계 : 20.1%

In [37]:
with open('naver.txt', 'r', encoding='UTF8') as f:
    lines = f.read().splitlines()
lines

['전도유망한 의과 레지던트가 어느 날 갑자기 좀비가 되어 생기는 이야기를 그린 드라마',
 '좀비 바이러스가 세상을 휩쓸고 간 3년 후, 바이러스 백신을 만들 것이라는 유일한 희망으로 죽음과 공포의 여정에 오른 사람들의 이야기를 그린 액션 호러 드라마',
 '엄마의 죽음으로 피아노를 칠 수 없게 된 천재 소년이 목표도 없이 지내던 중 친구의 들러리로 나가게 된 데이트로 인해 인생이 바뀌어 가는 이야기',
 '미국에서 가장 넓은 주인 알래스카의 울퉁불퉁한 해안에는 신기한 야생동물들이 서식하고 있다. 한겨울에 접어들어 해안이 어둠에 갇히기 전에 이곳의 동물들은 마음껏 햇살을 만끽한다. 위험천만한 알래스카 해안에서 펼쳐진 혹독한 환경을 이겨내며 살아가는 동물들을 만나본다.',
 '잠들지 않는 탐욕의 불빛들이 그 빛의 주인이 되기 위해 치열한 전쟁을 벌이는 과정을 그려낸 드라마',
 '인생의 한 순간 뜨거운 열풍에 휩싸인 주인공들의 사랑을 그린 치정 멜로드라마',
 '젊은 여교사가 서부의 작은 탄광촌에서 아이들을 가르치게 되면서 일어나는 이야기',
 '엉뚱하고 게으르지만 남다른 재치가 있는 잭과 아름답고 똑똑하지만 허당 기질이 다분한 조지나의 좌충우돌 수사 일기',
 '루이 14세가 왕정 확립을 위해 파리에서 베르사유로 권력을 이동시키려 하며 벌어지는 귀족들과의 갈등을 그린 드라마',
 '빚쟁이 실업자로 전락한 메이크업 아티스트와 아이돌 데뷔에 실패한 연습생이 만나 새로운 꿈을 찾아가는 로맨틱 코미디 드라마',
 '이스트 런던에 있는 가상의 도시에서 마약 거래와 거리 갱에 관여하는 젊은이들의 삶의 모습을 그린 드라마',
 '뉴욕 외곽도시의 한 고등학교를 배경으로 두 뱀파이어 형제와 인간 소녀의 로맨스를 그린 이야기',
 "특별하지 않지만 내 눈에는 반짝거리는 서로의 '제3의 매력'에 빠진 두 남녀가 스물의 봄, 스물일곱의 여름, 서른둘의 가을과 겨울을 함께 통과하는 연애의 사계절을 그릴 12년의 연애 대서사시",
 '핵폭발로 인해 문명이 붕괴된 지 천년 후 원시적인

In [38]:
sentences = [line for line in lines if line != '']

In [39]:
tagger = Komoran()
tagged_sentences = [tagger.pos(sent) for sent in sentences]

In [40]:
naver = []
for sent in tagged_sentences:    
    for word, tag in sent:
        if tag in ['VV', 'VA']:
            naver.append(word)
            
naver = [v+'다' for v in naver]

In [41]:
for sent in tagged_sentences:    
    for word, tag in sent:
        if tag in ['IC', 'NNP', 'MM', 'NP', 'NNG', 'VX', 'MAG', 'None', 'NR', 'NNB']:
            naver.append(word)

In [100]:
easy_count =0
for word in naver:
    if word in easy_words:
        easy_count+=1

In [105]:
mid_count =0
for word in naver:
    if word in mid_words:
        mid_count+=1

In [106]:
diff_count =0
for word in naver:
    if word in diff_words:
        diff_count+=1

In [103]:
naver

['되다',
 '생기다',
 '그리다',
 '휩쓸다',
 '만들다',
 '오르다',
 '그리다',
 '치다',
 '없다',
 '되다',
 '지내다',
 '나가다',
 '되다',
 '바뀌다',
 '가다',
 '넓다',
 '접어들다',
 '갇히다',
 '펼쳐지다',
 '이기다',
 '살아가다',
 '만나다',
 '잠들다',
 '되다',
 '위하다',
 '벌이다',
 '그리다',
 '뜨겁다',
 '휩싸이다',
 '그리다',
 '젊다',
 '작다',
 '가르치다',
 '되다',
 '일어나다',
 '게으르다',
 '남다르다',
 '있다',
 '아름답다',
 '위하다',
 '하다',
 '벌어지다',
 '그리다',
 '만나다',
 '새롭다',
 '찾아가다',
 '있다',
 '그리다',
 '그리다',
 '반짝거리다',
 '빠지다',
 '막다',
 '위하다',
 '그리다',
 '담다',
 '좋다',
 '만들다',
 '캐다',
 '들어가다',
 '갖다',
 '되다',
 '무찌르다',
 '많다',
 '징징대다',
 '하다',
 '싸우다',
 '하다',
 '맺다',
 '되다',
 '벌어지다',
 '그리다',
 '올라오다',
 '만나다',
 '오가다',
 '펼치다',
 '요동치다',
 '엇갈리다',
 '대하다',
 '꿈꾸다',
 '통하다',
 '전하다',
 '그만두다',
 '책임지다',
 '위하다',
 '찾아가다',
 '깨어나다',
 '놀랍다',
 '갖다',
 '되다',
 '맞서다',
 '싸우다',
 '도망치다',
 '벌어지다',
 '위하다',
 '보내다',
 '그리다',
 '모르다',
 '지키다',
 '위하다',
 '보내다',
 '강하다',
 '그리다',
 '벌어지다',
 '같다',
 '그리다',
 '그리다',
 '타고나다',
 '보다',
 '없다',
 '알다',
 '불타다',
 '만나다',
 '벌어지다',
 '느끼다',
 '숨기다',
 '파헤치다',
 '떨어지다',
 '잃다',
 '향하다',
 '돌아오다',
 '그리다',
 '지나다',
 '그리다',
 '되다',
 '담다',
 '

In [107]:
print(easy_count/len(naver))
print(mid_count/len(naver))
print(diff_count/len(naver))

0.32932045896054435
0.22046565930122397
0.19224475343789774


## 네이버
- 1단계: 32.9%
- 2단계: 22.0%
- 3단계: 19.2%

### 차이는 미세하지만, 넷플릭스가 네이버보다 1,2,3단계 단어 모두 더 많이 사용 
### 넷플릭스가 더 쉬운 단어 사용하는 경향